test/Deck-test.js

```diff
@@ -19,22 +20,18 @@ describe('Deck', function() {
     });
```

Comment:
At some point, I am going to have to do the beforeEach method for the test files.